lib/wipe-dependencies.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
```

Comment:
I think that's better to leave outside the repo. You often don't want to update everything too. I typically use [npm-check](https://www.npmjs.com/package/npm-check) which lets you pick and choose what you want to update.